cmd/eksctl-anywhere/cmd/flags.go

```diff
@@ -0,0 +1,39 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+)
+
+func addClusterFlags(cc *cobra.Command, clusterOpt clusterOptions, override ...string) {
+	overriddenFlags := make(map[string]struct{})
+	for _, flag := range override {
+		overriddenFlags[flag] = struct{}{}
+	}
+
+	if _, ok := overriddenFlags["filename"]; !ok {
+		cc.Flags().StringVarP(&clusterOpt.fileName, "filename", "f", "", "Filename that contains EKS-A cluster configuration")
+	}
+	if _, ok := overriddenFlags["bundles-override"]; !ok {
+		cc.Flags().StringVar(&clusterOpt.bundlesOverride, "bundles-override", "", "Override default Bundles manifest (not recommended)")
+	}
+	if _, ok := overriddenFlags["kubeconfig"]; !ok {
+		cc.Flags().StringVar(&clusterOpt.managementKubeconfig, "kubeconfig", "", "Management cluster kubeconfig file")
+	}
+	if _, ok := overriddenFlags["install-packages"]; !ok {
+		cc.Flags().StringVar(&clusterOpt.installPackages, "install-packages", "", "Location of curated packages configuration files to install to the cluster")
+	}
+}
+
+func bindAllFlags(cmd *cobra.Command, args []string) error {
```

Comment:
Moved the bindAllFlags to a common function between create, delete, upgrade. If defining them individually is intentional and we want to keep those separate I can revert out this section